package.json

```diff
@@ -50,11 +50,11 @@
     "angular-summernote": "0.8.1",
     "anythingslider": "https://github.com/CSS-Tricks/AnythingSlider/archive/v1.9.4.tar.gz",
     "backbone": "1.3.3",
-    "bootstrap": "3.3.4",
+    "bootstrap": "3.3.7",
```

Comment:
We are stuck on 3.3.4 since the bootstrap-rtl library stopped at 3.3.4 (this allows rlt to work (such as Hebrew language) in openemr when using the bootstrap package).
The next jump will be getting to bootstrap 4, which will require some analysis.